modules/nanopb/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021, Basalte bv
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_NANOPB)
+
+  set(NANOPB_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  include(${NANOPB_DIR}/extra/FindNanopb.cmake)
+
+  zephyr_library()
+
+  zephyr_include_directories(${NANOPB_DIR}/spm_headers/nanopb)
```

Comment:
This will not work on Windows, because it uses symlinks. Avoid the `spm_headers` folder completely and instead point to `modules/lib/nanopb`.

Reply:
Yes, reading your previous comment, I figured that would be the issue, will edit.